mdd/mdd.go

```
@@ -52,14 +53,15 @@ func (c *Containers) GetContainer(key int) *Container {
 func (c *Containers) LoadDefinition(definitions *dictionary.Dictionary) {
 	for i := range c.Containers {
 		container := &c.Containers[i]
-		definition, ok := definitions.Lookup(
+		definition, err := definitions.Lookup(
 			container.Header.Key,
 			container.Header.SchemaVersion,
 			container.Header.ExtVersion,
 		)
-		if ok {
-			container.LoadDefinition(definition)
+		if err != nil {
+			log.Errorf("Error lookup: %v", err)
```

return error instead of logging error